plugins/jobs/src/js/types/JobConnection.ts

```diff
@@ -1,19 +1,109 @@
-import { Job } from "#PLUGINS/jobs/src/js/types/Job";
+import { Job, JobTypeResolver } from "#PLUGINS/jobs/src/js/types/Job";
+import { JobResponse as MetronomeJobResponse } from "src/js/events/MetronomeClient";
 
+import JobStates from "#PLUGINS/jobs/src/js/constants/JobStates";
+import JobStatus from "#PLUGINS/jobs/src/js/constants/JobStatus";
 export interface JobConnection {
+  path: string[];
   filteredCount: number;
   totalCount: number;
   nodes: Job[];
 }
 
 export const JobConnectionSchema = `
 type JobConnection {
+  path: [String]!
   filteredCount: Int!
   totalCount: Int!
   nodes: [Job]!
 }
 `;
 
-export function JobConnectionTypeResolver(response: any): any {
-  return response;
+export interface JobsQueryArgs {
```

Comment:
should JobQueryArgs be its on type?

Reply:
I see your confusion, it was because of my bad wording.

I mean, the type JobsQueryArgs is found on the JobConnection.ts. I would try to look for it on JobsQueryArgs.ts (or d.ts).

Is there a reason for the type not to have its own file?

Reply:
Ah, fair point actually. And now as you say it, I remember that we wanted to change the foldername in the initial PR (but forgot to do so) 😕

Some History: When we created these files, Daniel wanted the folder `types` not named like that because of this exact reason. These files should define whatever type we use in GraphQL and contain all the necessary information related to it. I think we need to revisit and clarify this 🤔 @DanielMSchmidt, what is your opinion on this today?

That said, I dont think it is in scope of this PR.

Reply:
Let's do it in a cleanup JIRA